### DISCUSSION TITLE
How to change Mobile menu and Offcanvas icons

### DISCUSSION THREAD

**Discussion by nidzo80** [templaza/astroid-framework#150] [Q&A] [answered]
I want to replace default hamburger menu Mobile menu and Offcanvas icons using Font Awesome.  How can I achieve that?

**Comment by nidzo80** [comment, marked as answer]
To answer to my self.
Creating override of header folder and header.php located in /libraries/astroid/framework/frontend/
did the trick.

**Comment by Chacapamac** [comment]
@nidzo80 , can you tell me how exactly you create the override In Your Template —> html > ?
Thanks in Advance!

**Reply by nidzo80** [reply]
For example I use Astroid One template. From libraries/astroid/framework/frontend folder copy header.php file and paste it in
templates/astroid_template_one/html/frontend/

If frontend folder doesn't exist create it.
Also create folder called "header"

According to header style you chiose in Astroid settings from libraries/astroid/framework/frontend/header folder copy relevant file. There are horizontal.php, sticky php, sidebar php,stacked.php.
Paste that file in newly created header folder in template override (templates/astroid_template_one/html/frontend/header)
Then you can make changes. For example I changed offcanvas icon adding in stacked.php file this code
`<div><span style="color:#D00; font-size:29px;"><i class="fa-solid fa-cart-arrow-down"></i></span>
`
around line 64

![Screenshot_20220718_145851](https://user-images.githubusercontent.com/21330927/179516714-47155666-ae76-456d-90c8-0219edd6becc.jpg)

**Comment by Chacapamac** [comment]
Ok, I answer myself, the solution is indicated directly in the header.php file (Great idea to do that, Thanks Astroid!!!}
—> 	Just copy the file to ROOT/templates/YOURTEMPLATE/html/frontend/ folder to create and override

**Reply by nidzo80** [reply]
At the same time we were typing. Sorry for late.

**Reply by Chacapamac** [reply]
Thank You nidzo80 — Really cool that it is indicated in the file — Good Day!

Also, I find in my notes that you can do the same by styles of menu _(Horizontal,sidebar, stacked, sticky.php)_
Using —> /html/frontend/header/**YourHeaderStyle.php**
